src/Mod/Path/PathScripts/PathArray.py

```diff
@@ -75,39 +75,35 @@ def __setstate__(self, state):
         return None
 
     def setEditorProperties(self, obj):
+        if obj.Type == 'Linear1D':
+            Angle = Centre = CopiesX = CopiesY = SwapDirection = 2
+            Copies = Offset = 0
+        elif obj.Type == 'Linear2D':
+            Angle = Copies = Centre = 2
+            CopiesX = CopiesY = Offset = SwapDirection = 0
+        elif obj.Type == 'Polar':
+            Angle = Copies = Centre = 0
+            CopiesX = CopiesY = Offset = SwapDirection = 2
+
+        obj.setEditorMode('Angle', Angle)
```

Comment:
The variable names are ambiguous.  What's being set is the editor mode, not the angle/copy count ,etc.

Reply:
I will gladly revert to existing code if you like.

Reply:
The code is fine. It's more compact than the original which I appreciate.  But the variable name is confusing. When I see a variable name like 'Angle', I expect it to contain...a value that will be interpreted as an angle.  But in this case the variable 'Angle' contains a value that will be interpreted as an EditorMode.  So the variable name should be something like angleControlMode or angleControlVisibility.   Variable names is one place to _not_ be compact.  Be explicit.

Reply:
Variable names have been improved per indicated expectations.  Currently, 1 July 2021, builds are failing due to FEM unit tests.

Thanks for the feedback and oversight.